apps/mobile/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "mobile",
+  "version": "1.0.0",
+  "main": "index.ts",
+  "dependencies": {
+    "expo": "~47.0.12",
+    "expo-status-bar": "~1.4.2",
+    "react": "18.1.0",
+    "react-native": "0.70.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@types/react-native": "^0.71.1",
+    "babel-plugin-tsconfig-paths-module-resolver": "^1.0.4"
```

Comment:
should probably lock these down in mobile?

Reply:
I'me fine with patch upgrades happening automatically